build.gradle

```diff
@@ -71,12 +71,16 @@ static def checkForUpdate() {
         String availableBuildScript = new URL("https://raw.githubusercontent.com/SinTh0r4s/ExampleMod1.7.10/main/build.gradle").newInputStream(parameters).getText()
         String availableBuildScriptHash = getVersionHash(availableBuildScript)
         
-        if (currentBuildScriptHash.empty || availableBuildScriptHash.empty || currentBuildScriptHash == availableBuildScriptHash) return
 
-        println("Build script update available! Check https://github.com/SinTh0r4s/ExampleMod1.7.10/blob/main/build.gradle")
+        if (currentBuildScriptHash.empty || availableBuildScriptHash.empty || currentBuildScriptHash == availableBuildScriptHash)
+            return false
+
+        return true
     }
 }
-checkForUpdate()
+if(checkForUpdate()) {
+    println("Build script update available! Run 'gradle updateGTNHBuildScript'")
+}
```

Comment:
What do you think about adding `buildscriptAutoUpdate` flag to properties and if set to true updating things automatically?

Reply:
my only concern is if there will be ever a need to re-import the project. It should work normally as if you updated buildscript yourself (so a refresh should be sufficient), unless we migrate to `kts` at some point